### PR TITLE
Add missing track event source and type it properly

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Add missing source for the trackEvent.

**Why**
Otherwise it will error in prod env saying `Cannot sendAnalytics event: Error: Value trackEvent.source cannot be undefined`

**Added feature flags**
N/A

**Affected issues**  
ARC-1857

**How has this been tested?**  
N/A

**Whats Next?**
N/A

### CODE REVIEW

File: src/interfaces/common.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export enum AnalyticsTrackEventsEnum {
 }
 
 export enum AnalyticsTrackSource {
+	Default = "default",
```
Comment (Harminder84) [inline src/interfaces/common.ts:51]: why we need `default` tracks source?

Comment (gxueatlassian) [reply src/interfaces/common.ts:51]: Well, from the splunk log it seems we always need a source for TrackEvent. 
Some of this event can be for cloud or ghe, not sure what to put. So `Default` comes to my mind.

Comment (Harminder84) [reply src/interfaces/common.ts:51]: how about `gitHubCloud` instead of `default`?

Comment (gxueatlassian) [reply src/interfaces/common.ts]: Good point, I've updated to use cloud now.

**Review by Harminder84 — APPROVED** [review]

**Pull request merged**